readme.md

```diff
@@ -7,9 +7,6 @@ Validate Swedish social security numbers.
 ```dart
 import 'package:personnummer/personnummer.dart';
 
-Personnummer.valid(8507099805);
-//=> true
-
 Personnummer.valid('198507099805');
```

Comment:
Is this a real ssn?

Reply:
Maybe.. let's update it to something else :)

Reply:
Checked it and 9880-9999 is safe :)

Reply:
:+1: